lms/djangoapps/student_account/views.py

```diff
@@ -331,6 +331,8 @@ def get_user_orders(user):
         commerce_configuration, user, 'orders', api=api, querystring=user_query, cache_key=cache_key
     )
 
+    commerce_root = commerce_configuration.site or settings.ECOMMERCE_PUBLIC_URL_ROOT
```

Comment:
we need a way to use either otto or lms for the root.  So we can switch back and forth between the LMS hosted receipt page and the otto one.